requirements.txt

```diff
@@ -1 +1,2 @@
+connexion>=2.11.2,<3.0.0
```

Comment:
connexion released a new major version which has breaking changes. so had to restrict the connexion package limit to `<3.0.0`

Reply:
Can you please raise an issue for that in FOCA? Guess we should upgrade it at some point (but maybe not just right away...)

Reply:
Then after you can resolve this conversation and merge the PR :)

Reply:
raised an issue in foca - https://github.com/elixir-cloud-aai/foca/issues/182